stock_outgoing_shipment_report/models/stock_picking.py

```diff
@@ -18,12 +18,14 @@ def generate_stock_outgoing_shipment_report(self):
             product = move.product_id
             vals = {"move_id": move.id}
             if order:
-                carrier = order.carrier_id
+                carrier = self.carrier_id
```

Comment:
```suggestion
                carrier = order.carrier_id
```
With `self.carrier_id`, report generation for more than one delivery should end in a server error.

I'm thinking that maybe we should have taken the values for carrier_id and ship_service_id from stock.picking instead of sale.order but that requires some more tweaking on stock.picking.  Therefore we will stock to sale.order for now.

Reply:
Changed and confirmed operation.